src/components/LoginPage/Slide4.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImg1 from 'components/LoginPage/images/slide-4_img-1.svg';
+import slideImg2 from 'components/LoginPage/images/slide-4_img-2.svg';
```

Comment:
We could have more descriptive names: img-1 is a mind map. img-2 is google drive screenshot in a form of a cloud.

Also, please use only dashes in the filenames, no underscore pls.

Reply:
```
$ tree images/
images/
├── arrow-curve.svg
├── arrow.svg
├── icon-google.svg
├── slide-1.svg
├── slide-2.svg
├── slide-3_img-1.svg
├── slide-3_img-2.svg
├── slide-4_img-1.svg
├── slide-4_img-2.svg
└── slide-5.svg
```

In which slide is `arrow-curve` used?
What is the difference between `arrow` and `arrow-curve`?
The rest of the images are named with numbers only. If we use words as well, it would be easier to understand the code.
It is especially easy to confuse `slide-3_img-1.svg` and `slide-3_img-2.svg`: they are both displayed on one slide, so we can't tell where is which picture by looking at the code.

Reply:
`arrow-curve` is not used now. I merged it with an other svg picture. Sorry, forgot to delete.

Reply:
What do you think it would be better to rename - brain.svg or slide-1-brain.svg?

Reply:
I like the second option more 🙂

Reply:
Ok let's do `slide-1-brain.svg` :)
Another option is `slide-1/brain.svg`.

Reply:
Thanks!!! 😊

Reply:
Fixed in commit [cc616d8](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/cc616d87972f4b8b69277430dab419c1c1c0ae96)